commit b25883edde0ebc54a9fb2edea123772c79f5606c
Author: ashwin-h <ashwinh@vmware.com>
Date:   Tue Sep 1 17:47:17 2020 +0530

    Fix compilation issue in gptfdisk
    
    Details:
    https://svnweb.freebsd.org/ports/head/sysutils/gdisk/files/patch-gptcurses.cc?view=markup&pathrev=527130
    
    After update to 1.0.5, ncurses.h is incorrectly added with a macro.
    
    Signedoff-by: Ashwin H <ashwinh@vmware.com>

diff --git a/gptcurses.cc b/gptcurses.cc
index 1b18cf2..4ebfde1 100644
--- a/gptcurses.cc
+++ b/gptcurses.cc
@@ -23,11 +23,7 @@
 #include <iostream>
 #include <string>
 #include <sstream>
-#ifdef __APPLE__
 #include <ncurses.h>
-#else
-#include <ncursesw/ncurses.h>
-#endif
 #include "gptcurses.h"
 #include "support.h"
 
